ux.symfony.com/assets/styles/app.scss

```diff
@@ -26,16 +26,16 @@
 @import "../../vendor/twbs/bootstrap/scss/tables";
 @import "../../vendor/twbs/bootstrap/scss/forms";
 @import "../../vendor/twbs/bootstrap/scss/buttons";
-@import "../../vendor/twbs/bootstrap/scss/transitions";
-@import "../../vendor/twbs/bootstrap/scss/dropdown";
- @import "../../vendor/twbs/bootstrap/scss/button-group";
+// @import "../../vendor/twbs/bootstrap/scss/transitions";
```

Comment:
Why commented instead of removed?

Reply:
To keep track of what's in the standard distribution VS what we "removed" 

It's scss so it won't appear in compiled code, but i can clean this if you want, i know what the next step is ... and it won't require this file   👼

Reply:
(but don't know _when_)

Reply:
Oh I see, understood!